docs/ros-bridge.md

```diff
@@ -29,9 +29,12 @@ Velocity, orientation and acceleratoin information at 250hz.
 [Read all about the IMU model here.](imu.md)
 Data is in the `fsds/FSCar` frame.
 
-- `/fsds/odom` [nav_msgs/Odometry](https://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)   
+- `/fsds/for_testing/odom` [nav_msgs/Odometry](https://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)   
 Ground truth car position and orientation in NED frame. THIS WILL NOT BE STREAMED DURING COMPETITION.
 
+- `/fsds/track` [fsds_ros_bridge/Track](https://github.com/FS-Online/Formula-Student-Driverless-Simulator/blob/master/ros/src/fsds_ros_bridge/msg/Track.msg)   
+Ground truth cone position and color wrt starting location of the car. THIS WILL NOT BE STREAMED DURING COMPETITION.
```

Comment:
what is `wrt`?

Can document that this only publishes the cones that are part of the track spline? Any cones placed manually in the world are not published here.